.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
This is fine to have here, but you may also want to put it into a global gitignore file, since you're going to have these same issue on every repo you touch.